e2e/support/page-objects/AuthPage.ts

```
@@ -0,0 +1,68 @@
+
+  async isLoginFormVisible(): Promise<boolean> {
+    return await this.isElementVisible(this.emailInput);
+  }
+
+  async isLoggedIn(): Promise<boolean> {
+    return await this.isElementVisible(this.logoutButton);
+  }
+
+  get errorMessage(): Locator {
+    return this.errorMessage;
+  }
```

Biome (2.1.2)

[error] Duplicate class member name "errorMessage"



(lint/suspicious/noDuplicateClassMembers)